auction-server/api-types/src/opportunity.rs

```diff
@@ -312,6 +312,10 @@ pub enum OpportunityParamsV1ProgramSvm {
         #[serde_as(as = "DisplayFromStr")]
         user_wallet_address: Pubkey,
 
+        /// The user's current balance of the user token
+        #[schema(example = 10)]
+        user_mint_user_balance: u64,
```

Comment:
why does this matter? to prevent searcher from requesting too much in cases of searcher token specified?

Reply:
the reasoning is since we have this info, we might as well make it available to the searchers